lib/l10n/app_fr.arb

```diff
@@ -1,144 +1,134 @@
   },
   "username": "Nom d'utilisateur",
   "@username": {},
   "password": "Mot de passe",
   "@password": {},
-  "selectMusicLibraries": "Sélection des bibliothèques de musiques",
+  "logs": "Logs",
+  "@logs": {},
+  "next": "Prochain",
+  "@next": {},
+  "selectMusicLibraries": "Selectionner des bibliothèques",
   "@selectMusicLibraries": {
-    "description": "App bar title for library select screen"
+    "description": "Titre de l'écran de sélection de bibliothèque"
+  },
+  "couldNotFindLibraries": "Aucune bibliothèque trouvée",
+  "@couldNotFindLibraries": {
+    "description": "Cette erreur intervient lorsque l'utilisateur n'a pas de bibliothèque"
   },
-  "unknownName": "Nom inconnu",
+  "unknownName": "Nom Inconnu",
   "@unknownName": {},
   "songs": "Titres",
   "@songs": {},
-  "playlists": "Listes de lecture",
+  "albums": "Albums",
+  "@albums": {},
+  "artists": "Artistes",
+  "@artists": {},
+  "genres": "Genres",
+  "@genres": {},
+  "playlists": "Playlists",
   "@playlists": {},
+  "startMix": "Commencer un Mix",
+  "@startMix": {},
+  "startMixNoSongsArtist": "Appuyer longuement sur un artiste pour l'ajouter ou le retirer du mix avant de commencer",
+  "@startMixNoSongsArtist": {
+    "description": "Notification qui intervient quand l'utilisateur lance le mix instantané sans avoir sélectionné des artistes"
+  },
+  "startMixNoSongsAlbum": "Appuyer longuement sur un album pour l'ajouter ou le retirer du mix avant de commencer",
+  "@startMixNoSongsAlbum": {
+    "description": "Notification qui intervient quand l'utilisateur lance le mix instantané sans avoir sélectionné des albums"
+  },
+  "music": "Musique",
+  "@music": {},
   "clear": "Effacer",
   "@clear": {},
-  "settings": "Paramètres",
+  "favourites": "Favoris",
+  "@favourites": {},
+  "shuffleAll": "Tout mélanger",
+  "@shuffleAll": {},
+  "finamp": "Finamp",
+  "@finamp": {},
+  "downloads": "Téléchargements",
+  "@downloads": {},
+  "settings": "Réglages",
   "@settings": {},
+  "offlineMode": "Mode hors-ligne",
+  "@offlineMode": {},
   "sortOrder": "Ordre de tri",
   "@sortOrder": {},
-  "sortBy": "Trier par",
+  "sortBy": "Trié par",
   "@sortBy": {},
+  "album": "Album",
+  "@album": {},
+  "albumArtist": "Artiste de l'album",
+  "@albumArtist": {},
+  "artist": "Artiste",
+  "@artist": {},
+  "budget": "Budget",
+  "@budget": {},
+  "communityRating": "Note de la communauté",
+  "@communityRating": {},
+  "criticRating": "Note de la critique",
+  "@criticRating": {},
   "dateAdded": "Date d'ajout",
   "@dateAdded": {},
-  "datePlayed": "Date de dernière lecture",
+  "datePlayed": "Date de lecture",
   "@datePlayed": {},
-  "playCount": "Nombre de lectures",
+  "playCount": "nombre de lecture",
   "@playCount": {},
+  "premiereDate": "Date de première",
+  "@premiereDate": {},
+  "productionYear": "Date de production",
+  "@productionYear": {},
+  "name": "Nom",
+  "@name": {},
   "random": "Aléatoire",
   "@random": {},
+  "revenue": "Revenue",
```

Comment:
I think Recettes would be more appropriate

Reply:
Right!